lib/Dialect/BaseModelica/Transforms/EquationExpressionOpInterfaceImpl.cpp

```diff
@@ -1617,98 +2101,99 @@ void registerEquationExpressionOpInterfaceExternalModels(
                             BaseModelicaDialect *dialect) {
     // clang-format off
     // Equation root.
-    EquationSidesOp::attachInterface<::EquationSidesOpInterface>(*context);
+    ::EquationSidesOp::attachInterface<::EquationSidesOpInterface>(*context);
```

Comment:
I meant to add the anonymous namespace to the interface, which is now ok. But not also to the operation, which belongs to `mlir::bmodelica`. Please follow the original version of the code.